base.ini

```diff
@@ -5,6 +5,8 @@ retry.attempts = 3
 file_upload_bucket = snowflakes-files-dev
 file_upload_profile_name = snowflakes-files-upload
 elasticsearch.server = localhost:9201
+elasticsearch.shards.primary = 1
+elasticsearch.shards.replicate = 0
```

Comment:
@keenangraham @Bek I think we should start with adding the ability to configure the shards without actually changing it.

Reply:
With version 5, we don’t have the total shards limit and we can leave the default number as before. We should still change it to minimum for single machine demos, what do you think? I will look to it.

Reply:
I don't think it matters much on a single-node demo; they already index just as fast and they have a low search load.  What difference do you think it would make?